common/src/main/java/com/linecorp/centraldogma/internal/api/v1/ProjectDto.java

```diff
@@ -74,9 +96,9 @@ public String url() {
     }
 
     @Nullable
-    @JsonProperty("reposUrl")
```

Comment:
note: checked that this field doesn't seem to be used in the previous UI also, which guarantees backwards compatibility